core/jmx/api/src/main/java/org/opennms/netmgt/jmx/connection/AnyServerX509TrustManager.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
```

Comment:
This class needs to be refactored into ```core/lib/src/main/java/org/opennms/core/utils``` and reused everywhere. It looks like we have about 10 copies of this same code scattered everywhere.